### PR TITLE
Don't display the Symfony debug toolbar when printing the page

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #23303
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig [hunk 1]
```diff
@@ -527,3 +527,11 @@
     padding: 5px 0;
     margin-right: 10px;
 }
+
+/***** Media query print: Do not print the Toolbar. *****/
+@media print {
+    .sf-toolbar, .sf-error-toolbar {
+        display: none;
+        visibility: hidden;
```
Comment (ostrolucky) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig]: what's the reason for this if there is already `display: none`?

Comment (javiereguiluz) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig]: It was in the original CSS rule ... but you are right and this is no longer needed (Bootstrap v4 doesn't use that either. See https://github.com/twbs/bootstrap/blob/v4-dev/scss/_print.scss)

**Review by xabbuh — APPROVED** [review]

**Comment by stof** [comment]
Actually, the issue is that the media query hiding the whole toolbar (error one or normal one) is missing now.
the 2.7 branch has this code before the error toolbar:

```css
/***** Media query print: Do not print the Toolbar. *****/
@media print {
    .sf-toolbar {
        display: none;
        visibility: hidden;
    }
}
```

But the 2.8 branch does not seem to have it anymore.

I suspect a bad conflict resolution when merging branches

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig [hunk 2]
```diff
@@ -527,3 +527,10 @@
     padding: 5px 0;
     margin-right: 10px;
 }
+
+/***** Media query print: Do not print the Toolbar. *****/
+@media print {
+    .sf-toolbar, .sf-error-toolbar {
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig]: `.sf-error-toolbar` is useless here, as the error toolbar also has the `sf-toolbar` class

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**